default.yml

```diff
@@ -127,6 +134,15 @@ RSpec/Be:
 RSpec/DescribedClass:
   EnforcedStyle: explicit
 
+RSpec/EmptyLineAfterExampleGroup:
```

Comment:
For all these disabled rspec rules, API doesn't follow the convention. I tend to like them, but happy to respect the existing style.

Reply:
That's the old style in some of the older API code, I agree that we should now keep these whitespace lines enabled by default (and I know @timhaines and @djones strongly agree on this as well).

Can we remove these and keep them enabled, but silence them or auto-fix them in the API?